src/chart/generateCategoricalChart.tsx

```diff
@@ -1119,17 +1115,14 @@ export const generateCategoricalChart = ({
 
       this.clipPathId = `${props.id ?? uniqueId('recharts')}-clip`;
 
-      if (props.throttleDelay) {
-        this.triggeredAfterMouseMove = _.throttle(this.triggeredAfterMouseMove, props.throttleDelay);
-      }
+      // trigger 60fps
+      this.throttleTriggeredAfterMouseMove = _.throttle(this.triggeredAfterMouseMove, props.throttleDelay ?? 1000 / 60);
```

Comment:
why are we doing this if it isn't provided by the user? Does the 16ms make a difference? (this is a behavior breaking change even if unnoticeable, probably fine though)

Reply:
It was added for performance reasons. As you mentioned, it might not be noticeable to the average users, but it will significantly improve performance for users with large datasets.

Reply:
Even for the average user, other parts of the application can be impacted by performance issues, so having a default value might not be a bad idea (for now).